competition-simulator.md

```diff
@@ -0,0 +1,131 @@
+2. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+
+You may receive a warning about your computer's GPU not being good enough, which can be ignored.
+
+#### Changing your version of Python
+
+If webots is picking up the incorrect version of Python, you'll need to change it. This can be done using `Tools > Preferences > General > Python command`. You'll need to ensure a matching version of Python is installed.
+
+## Overview
+
+Within the IDE, there are a few different panels:
+
+- In the centre of your screen is the 3D simulated view of the arena
+- On the left is a tree hierarchy of all elements in this "world"
+- On the right is your text editor for modifying the `sr_controller` code for example
+- At the bottom is your console
+- At the top are your general controls which include the time controls. Press the centre play button to run the simulation at normal speed.
+
+### Useful links
+
+- [Camera Controls](https://www.cyberbotics.com/doc/guide/the-3d-window#navigation-in-the-scene)
+- [Graphics settings](https://www.cyberbotics.com/doc/guide/preferences#opengl) (Useful if Webots is running slowly)
+
+## Developing your code
+
+On first run, the robot will execute an example program for convenience. On first run, this will be copied to the directory `competition-simulator` is stored in:
+
+```
+.
+├── competition-simulator
+│   ├── ...
+│   └─ worlds
+│       └── Arena.wbt
+└── robot.py
+```
+
+Your code should be developed in `robot.py`.
+
+
+<div class="warning">
+  Only your controller code will be present in the competition environment.
+</div>
+
+<div class="warning">
+  You will be responsible for backing up and versioning your code, and collaborating with your fellow team members. The <a href="https://studentrobotics.org/ide/">IDE</a> is unsuitable for this simulator.
+</div>
+
+### Updates
+
+Occasionally, we may release an update to the simulation. To update, you will need to delete the `competition-simulator-master` directory, and re-download it using the above link.
+
+## Robot
+
+![]({{ site.baseurl }}/resources/competition-simulator/robot-front.png)
+
+## Programming Interface
+
+Unless otherwise stated, the simulator’s API is the same as the real SR API described in the [programming docs]({{ site.baseurl }}/programming/).
+
+### Motors
+
+Your robot has 2 motor boards attached, each with 2 motors. Board `0` has the left wheel in port `m0`, and the right wheel in `m1`. Board `1` has the gripper lift motor in `m0`, and the finger motors in `m1`.
+
+### Ruggeduino
+
+Your robot has 5 microswitches and 6 distance sensors, attached to the digital and analogue pins respectively. These are all attached to a single ruggeduino.
+
+Because these sensors are pre-attached to the ruggeduino, you do not need to set its `pin_mode`.
+
+#### Microswitches
+
+The microswitches are attached to digital pins 2-6:
+
+- Front
+- Back
+- Between gripper fingers
+- Left gripper finger
+- Right gripper finger
```

Comment:
I think we should explicitly mention which pin each of these switches  belong to.
```suggestion
- Front - Pin  2
- Back - Pin 3
- Between gripper fingers - Pin 4
- Left gripper finger - Pin 5
- Right gripper finger - Pin 6
```

Reply:
Actually maybe a table would look better with Pin Number and Location being the headers.